tfe/resource_tfe_team_project_access_test.go

```diff
@@ -0,0 +1,143 @@
+package tfe
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccTFETeamProjectAccess_admin(t *testing.T) {
+	skipUnlessBeta(t)
+
+	tmAccess := &tfe.TeamProjectAccess{}
+	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+
```

Comment:
We're going to need elevated privileges here, so `createBusinessOrganization()` :smile: